eodag/utils/__init__.py

```diff
@@ -1541,3 +1541,20 @@ def dict_md5sum(input_dict: dict[str, Any]) -> str:
     >>> assert(dict_md5sum({"a": 4, "b": {"b": 3, "c": 1, "a": 2}}) == hd)
     """
     return obj_md5sum(sort_dict(input_dict))
+
+
+def remove_str_array_quotes(input_str: str) -> str:
+    """Remove quotes around arrays to avoid json parsing errors
```

Comment:
```suggestion
    """Remove quotes around arrays to avoid json parsing errors

    :param input_str: string to format
    :returns: string without quotes surrounding array brackets

```